Shared/Features/Freaks/View/FreaksButtonsView.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftUI
+
+struct FreaksButtonsView: View {
+    var body: some View {
+        HStack(spacing: 0) {
+            Button("Skills") {
+                print("skills")
+            }
+            .frame(maxWidth: .infinity)
+            .frame(height: 60)
+            Rectangle()
+                .frame(width: 3, height: 60)
+
+            Button("Projects") {
+                print("projects")
+            }
+            .frame(maxWidth: .infinity)
+            .frame(height: 60)
+        }
+        .foregroundColor(.white)
```

Comment:
Instead of `.white` let's use `SecondaryColor`. This will help us a lot if we will change the app theme colors later.